### PR TITLE
Task4 database

### PR DESCRIPTION
**Pull request by ArtiFisher**
[Task 4](https://github.com/rolling-scopes-school/nodejs-aws-tasks/blob/main/task4-rds/task.md)
All tasks are done, including optional ones.

URL for POSTing(and GETting) products: https://g86uhrmyl7.execute-api.eu-west-1.amazonaws.com/dev/products
Parameters should be added to body of request, "title" is mandatory, other fields will use default values.

[App on CloudFront](https://d2q4x5i02o5bbw.cloudfront.net/)
[Products list](https://g86uhrmyl7.execute-api.eu-west-1.amazonaws.com/dev/products)

Also fixed FE inconsistencies([PR](https://github.com/ArtiFisher/nodejs-aws-fe/pull/3)) to make adding products possible via [Create new product form](https://d2q4x5i02o5bbw.cloudfront.net/admin/product-form/)

### CODE REVIEW

File: db-scripts/insert-into-stocks.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+insert into stocks (product_id, count) values
+('9cab85ac-a4c6-4fdb-9961-d75946c8e3f1', 5),
+('1a2438c4-f653-4dce-8fca-84c29ad9ab5a', 4),
+('58375bd3-30f8-4e07-9ccb-a6b87bd71832', 6),
+('cbb1aff4-7555-4a0f-8632-36c5595a4360', 2),
+('23f37489-e08d-4f10-bcba-05862f595dac', 0),
+('0ded6690-f7cc-4e8c-86e8-532eb413440c', 2),
+('2837c292-9eae-4fe2-9310-730a2572f8a8', 1),
+('c250ad12-d11b-4c3a-b6ce-59d1f62267eb', 3);
```
Comment (lithiumfallen) [inline db-scripts/insert-into-stocks.sql:9]: To fill data with 2 related data better to use `WITH` operator. To pass id from first table into second and avoid passing manual uuid.

File: product-service/handlers/getProductsById.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    statusCode = StatusCodes.INTERNAL_SERVER_ERROR;
+  }
+  finally {
+    client.end();
+  }
+  return {
+    headers: {
+      'Access-Control-Allow-Origin': '*',
+      'Access-Control-Allow-Credentials': true,
+    },
```
Comment (lithiumfallen) [inline product-service/handlers/getProductsById.js:34]: You can create a common headers for all functions to avoid repeating this code.

Comment (ArtiFisher) [reply product-service/handlers/getProductsById.js:34]: Thanks, will do!

File: db-scripts/insert-into-stocks.sql (hunk 1 above)
Comment (ArtiFisher) [reply db-scripts/insert-into-stocks.sql:9]: It's a script that is not expected to be used at all if everything is working fine. So I figured there's no point in optimizing it :)

**Pull request merged**